### ISSUE TITLE
Sending information other than errors 

### ISSUE THREAD

**Issue by chvvkumar** [daleghent/nina-ground-station#11]
Hello! 

I am wondering if it is possible to send other information using ground station. Things like HFR, RMS, sequence progress status etc. I looked for documentation to see if this was possible but my Google Fu is failing me.

### Use case:
I am looking for a way to send the following information after a sequence or image over MQTT to my Home Assistant instance. Home Assistant will in turn display this as a dashboard for how the imaging run is progressing

- Target (Name, coordinates, time remaining above horizon etc.)
- Frame type (light/fllat/darkflat etc.)
- Exposure details (duration, gain etc.)
- Sequence details (Current imaging parameters, remaining time on target etc.)
- Equipment status 
  - Focuser position
  - Mount: Park/Home position
   
Basically what this project is aiming to do but within NINA and not ASCOM (as some ZWO cameras work better with native drivers over ASCOM).

**Comment by chvvkumar** [comment]
Found the information I was looking in the 'Message token help' section of the plug-in settings. Closing this issue. Thanks for your helpp @daleghent
